lib/reckon/app.rb

```diff
@@ -66,18 +67,21 @@ def learn!
 
     def learn_about_account(account, data, parse_regexps = false)
       accounts[account] ||= 0
-      if parse_regexps && data.start_with?('/')
+      if data.respond_to?(:to_h)
+        pattern = (parse_regexps ? try_regexp(data["Pattern"]) : nil) || data["Pattern"]
+        return if pattern.nil?
+        counterpart = (parse_regexps ? try_regexp(data["Account"]) : nil) || data["Account"]
+        complex_tokens << {
+          pattern: pattern,
+          counterpart: counterpart,
+          account: account,
+          skip: data["Skip"] == true
+        }
+      elsif parse_regexps && data.start_with?('/')
```

Comment:
Do you think it makes sense to move everything to complex tokens, adding these to `complex_tokens` as well, instead of having `complex_tokens` and `regexps`?